devops/find_duplicated_fn_keys/main.py

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/python
+
+'''
+
+Purpose:    to detect if there is a duplicated wallet address or BLS Key
+Author:     Andy Wu (andy@harmony.one)
+Date:       Oct 17, 2019
+
+Usage:
+$python3 main.py <file_path> <last_number_lines>
+
+Example:
+$python3 main.py /Users/bwu2/go/src/github.com/harmony-one/harmony/internal/genesis/foundational.go 320
+based on the assumption we have and only have 320 FN keys to verify
+'''
+
+import sys
+import re
+
+if len(sys.argv) != 3:
+    print("Usage: $python3 main.py <filepath> <nlines>")
+    sys.exit(1)
+
+fname, nlines = sys.argv[1:]
+num_lines = int(nlines)
+
+array_epoch_string = []
+
+array_addr = []
+array_bls = []
+
+def tail(file, num_lines=num_lines):
+    with open(file) as f:
+        content = f.read().splitlines()
+
+    count = len(content)
+    for i in range(count-int(num_lines)-1, count-1):
```

Comment:
This line confusing, write comment explaining semantic intent

Reply:
The reason for that is the var variable has an ending `}`, this `-1` is simply exclude the last line. Yes, it would be better to leave some comments there.